ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/base/v1/base/clientlibs/editor/js/editDialog.js

```diff
@@ -25,7 +25,12 @@
         BASE_ENUMNAMES_VISIBLE = ".cmp-adaptiveform-base__enumNames",
         BASE_ENUMNAMES_HIDDEN = ".cmp-adaptiveform-base__enumNamesHidden",
         BASE_ASSISTPRIORITY_CUSTOMTEXT = ".cmp-adaptiveform-base__assistpriority-customtext",
-        BASE_DORBINDREF = ".cmp-adaptiveform-base__dorBindRef",
+        BASE_TITLE = ".cmp-adaptiveform-base__title",
+        BASE_RICH_TEXT_TITLE = ".cmp-adaptiveform-base__richtexttitle",
+        BASE_WRAPPER_RICH_TEXT_TITLE = "input[data-wrapperclass='cmp-adaptiveform-base__richtexttitle']",
```

Comment:
Selector should not have input

Reply:
rte saves the same class in two different places and we need to get all two of them.
1. In actual input of the form which is hidden.
2. In RTE div, where user actually writes the value